README.md

```diff
@@ -1 +1,27 @@
 # node-red-desktop
+The Node-RED desktop is a desktop application which includes Node-RED, Node.js, nodes, and Chromium browser engine.
```

Comment:
This needs to make it clear this project is under development and **not** ready for general use. The Quick Start instructions are not valid until we do a release.